### PR TITLE
 Trying to come up with a better description

### PR DESCRIPTION
**Pull request by karasuhebi**
I'm not totally in love with the description I wrote there (I feel like it could be made A LOT shorter) but I wanted to at least open a PR to get the discussion started. Let's hear some ideas!

Some context: My goal here is to get the description to really get Steam Deck owners who use Moonlight to understand what exactly MoonDeck brings to the table/makes it unique and why they would be interested in that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # MoonDeck ![Status](https://github.com/FrogTheFrog/moondeck/actions/workflows/build.yaml/badge.svg) [![Chat](https://img.shields.io/badge/Chat-on%20discord-7289da.svg)](https://discord.com/invite/U88fbeHyzt) ![Decky store](https://img.shields.io/badge/dynamic/json?color=blue&label=Decky%20version&query=%24%5B%3F%28%40.name%3D%3D%27MoonDeck%27%29%5D.versions%5B0%5D.name&url=https%3A%2F%2Fplugins.deckbrew.xyz%2Fplugins)
 
-A plugin that makes it easier to manage your gamestream sessions from the SteamDeck.
+A plugin that lets you play any of your Steam games via Moonlight without needing to add them to Sunshine first, providing a similar experience to GeForce GameStream or Steam Remote Play
```
Comment (FrogTheFrog) [inline README.md:3]:
```suggestion
A plugin that lets you play any of your Steam games via Moonlight without needing to add them to Sunshine first, providing a similar experience to GeForce GameStream or Steam Remote Play.
```

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "moondeck",
   "version": "1.7.2",
-  "description": "MoonDeck is an automation tool that will simplify launching your Steam games via the Moonlight client for streaming.",
+  "description": "MoonDeck lets you play any of your Steam games via Moonlight without needing to add them to Sunshine first, providing a similar experience to GeForce GameStream or Steam Remote Play",
```
Comment (FrogTheFrog) [inline package.json:4]:
```suggestion
  "description": "MoonDeck lets you play any of your Steam games via Moonlight without needing to add them to Sunshine first, providing a similar experience to GeForce GameStream or Steam Remote Play.",
```

File: plugin.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
   "api_version": 1,
   "publish": {
     "tags": ["gamestream", "moonlight", "moondeck", "streaming"],
-    "description": "MoonDeck is an automation tool that will simplify launching your Steam games via the Moonlight client for streaming.",
+    "description": "MoonDeck lets you play any of your Steam games via Moonlight without needing to add them to Sunshine first, providing a similar experience to GeForce GameStream or Steam Remote Play",
```
Comment (FrogTheFrog) [inline plugin.json:8]:
```suggestion
    "description": "MoonDeck lets you play any of your Steam games via Moonlight without needing to add them to Sunshine first, providing a similar experience to GeForce GameStream or Steam Remote Play.",
```

**Comment by FrogTheFrog** [comment]
I actually like the description. It's a lot better than mine!

**Comment by FrogTheFrog** [comment]
Do keep in mind that the next MoonDeck release will be only after the Decky v3 has been released. Since the plugin has already been adapted to work with it.

**Comment by karasuhebi** [comment]
> I actually like the description. It's a lot better than mine!

Thanks! I'm glad to hear you like it. I like it as well but I was hoping to make it shorter, unless you don't have a problem with the length in which case feel free to approve and merge the PR.

**Comment by karasuhebi** [comment]
I just realized that the About for this repo still uses the old text of "A plugin that makes it easier to manage your gamestream sessions from the SteamDeck."

I don't think I can PR that. Would you mind updating that to the new string of "A plugin that lets you play any of your Steam games via Moonlight without needing to add them to Sunshine first, providing a similar experience to GeForce GameStream or Steam Remote Play."

**Pull request merged**